### PR TITLE
poppler, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Needs merge for poppler-data first: https://github.com/haikuports/haikuports/pull/6510

**Comment by Begasus** [comment]
Packages envolved (some probably needs revbump):
- [ ] booksorg
- [ ] qpdfview
- [ ] texstudio (got a recipe ready to bump to the latest version (no PR there yet)
- [ ] hippo
- [ ] yacreader
- [ ] callibre
- [x] extractpdmark (no revbump needed)
- [ ] scribus
- [ ] krita
- [ ] okular
- [ ] openscenegraph
- [ ] calligra
- [ ] libreoffice

**Comment by Begasus** [comment]
@korli could this be merged? I need this merge for an updated TeXstudio (4.2.1 atm), this needs to be reviewed after nss is fixed (https://github.com/haikuports/haikuports/pull/6511)

File: app-text/poppler/poppler-21.12.0.recipe [hunk 1]
```diff
@@ -255,10 +253,13 @@ PROVIDES="
...
 	cmd:pdftotext
 	cmd:pdfunite
+	lib:libpoppler$secondaryArchSuffix = $libVersionCompat
+	lib:libpoppler_cpp$secondaryArchSuffix = $libCppVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	poppler_data >= 0.4.10
+	poppler_data >= 0.4.11
+#	nss${secondaryArchSuffix}_devel
```
Comment (korli) [inline app-text/poppler/poppler-21.12.0.recipe:262]: Why this comment here?

Comment (Begasus) [reply app-text/poppler/poppler-21.12.0.recipe:262]: Probably a left over trying to fix the issue, can be removed

**Comment by Begasus** [comment]
Bumped to latest version

**Comment by korli** [comment]
Looks good!

**Comment by Begasus** [comment]
For build failure on libreoffice maybe these patches could fix the problems:
32bit:
https://cgit.freedesktop.org/libreoffice/core/commit/?h=libreoffice-7.1.0.1&id=f6112bf4a0577db4c43370aafa37a073e73a94b1

64bit:
https://git.libreoffice.org/core/+/abeb3847b290c46138960933f18362c5b8a95c17%5E%21

**Pull request merged**